src/api/procedures/modifyAllowance.ts

```diff
@@ -0,0 +1,143 @@
+import BigNumber from 'bignumber.js';
+
+import { PolymeshError, Procedure, Subsidy } from '~/internal';
+import { ErrorCode, TxTags } from '~/types';
+import { AllowanceOperation, ProcedureAuthorization } from '~/types/internal';
+import { bigNumberToBalance, stringToAccountId } from '~/utils/conversion';
+
+export interface IncreaseAllowanceParams {
+  /**
+   * amount of POLYX to be increased
```

Comment:
```suggestion
   * amount of POLYX to be increase the allowance by
```

nit, but the phrasing seems a weird to me without a noun / subject